go.mod

```diff
@@ -139,3 +139,5 @@ require (
 	gopkg.in/validator.v2 v2.0.1 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/grpc-ecosystem/grpc-gateway => github.com/temporalio/grpc-gateway v1.17.0
```

Comment:
Which dependency is forcing this explicit `replace`? I just want to confirm we cannot replace this dependency upstream.

Reply:
We may also need to make a note about this override for library users who are depending on Temporalite through go mod.

Reply:
the initial issue came from https://github.com/sveltejs/kit/issues/3069

`grpc-gateway` v2 introduced an option `WithUnescapingMode` that would allow to address the escaping properly, however this was not moved into v1. See https://github.com/grpc-ecosystem/grpc-gateway/issues/2629 for context and why we had to fork

Reply:
also considered upgrading to grpc-gateway v2, asked the server team if we could move to [protobuf-apiv2](https://go.dev/blog/protobuf-apiv2). In short such upgrade didn't get into our timeline yet because of many moving pieces

Reply:
added a readme comment

Reply:
That's unfortunate but understandable.

Reply:
@feedmeapples is there an existing issue for migrating to the v2 protobuf libraries we can link here?

Reply:
https://github.com/temporalio/temporal/issues/38